src/components/InputDataDialog.js

```diff
@@ -0,0 +1,63 @@
+import {nanoid} from 'nanoid';
+import styled from 'styled-components';
+
+export default function InputDataDialog({setInputs, inputs}) {
+    let actualDate = new Date();
+    let formattedActualDate = actualDate.toLocaleDateString()
+
+    return (
+        <FormInputData onSubmit={handleSubmit}>
+            <label htmlFor="inputfield">Please insert data - only numbers allowed:</label>
+            <input
+                type="number" min="0"
+                id="inputfield" name="inputfield" maxLength={20} required />
+            <button>Submit</button>
+        </FormInputData>
+
+    );
+
+    function handleSubmit(inputEvent) {
+        inputEvent.preventDefault();
+        const inputData = inputEvent.target;
+        const inputDataValue = inputData.elements.inputfield.value.trim();
+        
+        
+        const newInput = {
+        id: nanoid(),
+        date: formattedActualDate, 
+        value: inputDataValue,
+        };
+
+        setInputs([newInput, ...inputs]);
```

Comment:
Here you can call the function which you pass down from the `App.js` to update your state :) instead of calling `setInputs`